vista-fhir-query/src/test/java/gov/va/api/health/vistafhirquery/service/controller/coverageeligibilityresponse/R4CoverageEligibilityResponseToVistaFileTransformerTest.java

```
@@ -32,6 +33,30 @@ private R4CoverageEligibilityResponseToVistaFileTransformer _transformer() {
         .build();
   }
 
+  @Test
+  void extractFromCodeableConceptOrDie() {
+    assertThatExceptionOfType(MissingRequiredField.class)
+        .isThrownBy(() -> _transformer().extractFromCodeableConceptOrDie("", "", "", "", null));
```

Not sure how I feel about this. I would expect fake values exception for the one you're using to cause the error